AutoCollection/ServerRequests.ts

```diff
@@ -18,6 +19,7 @@ class AutoCollectServerRequests {
     private _client:Client;
     private _isEnabled: boolean;
     private _isInitialized: boolean;
+    private _isAutoCorrelating: boolean;
```

Comment:
Its on by default. Works correctly on Breeze. I don't have numbers for perf hit (you would need quite a bit of synthetic traffic to really measure I would presume). We can try in INT.